.github/workflows/ci.yaml

```diff
@@ -45,17 +44,13 @@ jobs:
         #  https://github.com/giggio/node-chromedriver - explanation
         #  https://sites.google.com/a/chromium.org/chromedriver/downloads - available versions
         export CHROMEDRIVER_VERSION=75.0.3770.140
+        yarn
```

Comment:
This would run yarn and then export a variable and yarn again. Isn't it redundant or what did I miss?

Reply:
The idea was to have separate `yarn` steps for windows and for rest.

Previously, `yarn` was started in `pretest` step which was kind-a fishy, as it should be ran in "Install dependencies" step.

Moreover, "Install deps ..(Windows)" was de-facto no-op as onlyset env variable only for shell which immediately exits at end of step.

So, it looks like CHROMEDRIVER_VERSIOS is not needed at all, so i'll unify install dependencies ...